lib/app/services/api.dart

```diff
@@ -44,6 +48,67 @@ class ApiService {
     }).toList();
   }
 
+  Future<List<Course>> listCourses() async {
+    var response = await network.get('courses?');
```

Comment:
Do we need the question mark here?

Reply:
The question mark can be removed. It was added to reflect the way news are loaded, so it could be removed there as well.